### PR TITLE
chore(FX-4178): cleanup trending sort AB test

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Chore**

This PR solves [FX-4178]

### Description

- [x] Cleans up trending sort experiment

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FX-4178]: https://artsyproduct.atlassian.net/browse/FX-4178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ describe("ArtistArtworkFilter", () => {
       const option = screen.getByRole("option", { name: "Default" })
 
       expect(option).toBeInTheDocument()
-      expect(option).toHaveValue("-default_trending_score")
+      expect(option).toHaveValue("-decayed_merch")
```
Comment (dimatretyak) [inline src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx:120]: **suggestion**: i think you can remove [all this case](https://github.com/artsy/force/blob/2dbc28db82bb187a92a54190633c3d5a2f256649/src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx#L78)

**Comment by dimatretyak** [comment]
and all [this code](https://github.com/artsy/force/blob/main/src/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts#L19-L25)

File: src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx (hunk 1 above)
Comment (gkartalis) [reply src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx:120]: 🙈 you're definitely right

Comment (gkartalis) [reply src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx:120]: 🙈 you're definitely right

**Comment by gkartalis** [comment]
> and all [this code](https://github.com/artsy/force/blob/main/src/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts#L19-L25)

done 👌

**Comment by dimatretyak** [comment]
> wait, about this, should I also remove [this](https://github.com/artsy/force/blob/172e29a18e301f0be87099d526d6cd3d9c46e4ec/src/Apps/Artist/artistRoutes.tsx#L128) line? 🤔

nope 😅

**Review by dimatretyak — APPROVED** [review]

**Pull request merged**